exercises/atbash-cipher/example.rkt

```diff
@@ -0,0 +1,34 @@
+#lang racket
+
+(provide (contract-out
+          [encode (string? . -> . string?)]
+          [decode (string? . -> . string?)]))
+
+(define (prepare-string m)
+  (filter (λ(s)
```

Comment:
I don't think this is a must, but if you want to have access to nice tools like `string-filter` you can use `srfi-13`

Reply:
Frankly, I am not as familiar with the SRFI's as I should be. I'm sure that it is more elegant to use the function from SRFI 13. I'll probably have another commit later today.

I really appreciate the code review!

Reply:
No problem. I started messing with this, and given that there is no simple string chunking function it is actually more convenient to keep it as a list until the end.

I started working on a refactored version off of your branch.  I will probably make a PR to your branch later, so you can see my alternative solution, but you are under no obligation to override your work with mine!  I mostly just think you will find it interesting to see another approaches :smile:

Reply:
Just bumping this.  @guygastineau are you happy if this gets merged are you OK with that? @cousinitt also are *you* happy for this to get merged?

Reply:
It is fine with me

Reply:
Hi @robertpostill - I am fine with this being merged. The delay was mostly me mulling over the currying, which was something that I had not seen much of in Racket. Since you were bumping this, I will merge it in my branch for sure.